src/types/weakref/proxy.rs

```diff
@@ -22,6 +22,8 @@ pyobject_native_type_named!(PyWeakrefProxy);
 
 impl PyTypeCheck for PyWeakrefProxy {
     const NAME: &'static str = "weakref.ProxyTypes";
+    #[cfg(feature = "experimental-inspect")]
+    const PYTHON_TYPE: &'static str = "weakref.ProxyType | weakref.CallableProxyType";
```

Comment:
I wonder whether there's ever a way that we could get these type names from the Python interpreter, by launching an interpreter at build time and importing them all. That's probably a complicated mess best avoided for now when hard-coding is likely to be good enough.

Reply:
Yes! I would love to avoid running the Python interpreter when building type stubs to allow easy cross-compilation.